### DISCUSSION TITLE
Provide XML Documentation and publish `*.snupkg`

### DISCUSSION THREAD

**Discussion by samtrion** [thomhurst/TUnit#1424] [Ideas]
Please provide qualified xml summaries for all common methods and publish the `*.snupkg` packages. This simplifies the integration or migration from other test frameworks significantly, furthermore it offers a first-class integration in existing IDEs.

**Comment by thomhurst** [comment]
snpkgs are published: https://www.nuget.org/api/v2/symbolpackage/TUnit/0.5.6

**Reply by samtrion** [reply]
That's strange, VS must be a bit picky today.

**Reply by samtrion** [reply]
Do you need help with extending the XML Summaries?

**Reply by thomhurst** [reply]
If you're free to help out sure :)
